ml_enabler/models/ml_model.py

```diff
@@ -34,6 +34,11 @@ def get_tiles_by_quadkey(prediction_id: int, quadkeys: tuple, zoom: int):
         return db.session.query(func.substr(PredictionTile.quadkey, 1, zoom).label('qaudkey'), func.avg(cast(cast(PredictionTile.predictions['ml_prediction'], sqlalchemy.String), sqlalchemy.Float)).label('ml_prediction')).filter(PredictionTile.prediction_id == prediction_id).filter(func.substr(PredictionTile.quadkey, 1, zoom).in_(quadkeys)).group_by(func.substr(PredictionTile.quadkey, 1, zoom)).all()
 
 
+    @staticmethod
+    def get_tiles_in_polygon(prediction_id: int, polygon: str):
```

Comment:
@geohacker could we rename this function? It was a bit confusing reading through since it doesn't exactly `get_tiles_in_polygon` but maybe something like `get_aggregate_for_polygon` ?

In general, this PR looks great.